src/check.c

```diff
@@ -277,6 +277,11 @@ uint32_t alloc_found_entry(DOS_FS *fs, int force_create)
     if (!dir || force_create) {
         DIR_ENT de = {0, };
         DOS_FILE *walk = NULL;
+        struct tm *ctime;
+        time_t current;
+
+        time(&current);
+        ctime = gmtime(&current);
```

Comment:
Could you explain why replacing localtime() with gmtime()?
And it is better to call gmtime_r instead of gmtime

Reply:
Sorry, please ignore above comments. according to fat32 specification, time fields should be saved with localtime.
I think it's little bit wierd. but I'll modify it using localtime().